bug: AssertionError: (None, <Day>) pada uji `test__dataframe_table`

### Sistem
* Versi hidrokit: 0.3.7
* Distribusi python: 3.6.13
* Sistem operasi: Windows

### Ringkasan Permasalahan

Pada saat testing, hasil dari uji `test_prep_excel.test__dataframe_table` menghasilkan error `AssertionError: (None, <Day>)`. Berdasarkan `pandas-dev/pandas#35446`, kemungkinan error terjadi saat _parsing_ tanggal.

### Error
```python
./tests/test_prep_excel.py::test__dataframe_table Failed: [undefined]AssertionError: (None, <Day>)
def test__dataframe_table():
        filepath = 'tests/data/excel/2013 DUGAAIR.xls'
        pivot = excel._file_single_pivot(filepath, template='pdderi')
        dataframe = excel._dataframe_table(pivot, 2013, name='dugaair')
    
        result = pd.read_csv('tests/data/2013 DUGAAIR.csv',
                             index_col=0, parse_dates=True)
        assert_frame_equal(
>           result, dataframe
        )
E       AssertionError: (None, <Day>)
```

### Solusi

solusi sementara menambah argumen `check_like=True` pada fungsi `assert_frame_equal(...)`

--- a/tests/test_prep_excel.py
+++ b/tests/test_prep_excel.py
@@ -64,5 +64,5 @@ def test__dataframe_table():
     result = pd.read_csv('tests/data/2013 DUGAAIR.csv',
                          index_col=0, parse_dates=True)
     assert_frame_equal(
-        result, dataframe
+        result, dataframe, check_like=True
     )

